plugins/base/src/main/kotlin/parsers/Parser.kt

```diff
@@ -49,10 +51,51 @@ abstract class Parser {
 
     private fun jkdocToListOfPairs(javadoc: String): List<Pair<String, String>> =
         "description $javadoc"
-            .split("\n@")
+            .splitIgnoredInsideBackticks("\n@")
             .map { content ->
                 val contentWithEscapedAts = content.replace("\\@", "@")
                 val (tag, body) = contentWithEscapedAts.split(" ", limit = 2)
                 tag to body
             }
+
+    private fun CharSequence.splitIgnoredInsideBackticks(delimiter: String): List<String> {
+        var countOfBackticks = 0
+        var countOfBackticksInOpeningFence = 0
+
+        var isInCode = false
+        val result = mutableListOf<String>()
+        var rangeStart = 0
+        var rangeEnd = 0
+        var currentOffset = 0
+        while (currentOffset < length) {
+
+            if (get(currentOffset) == '`') {
+                countOfBackticks++
```

Comment:
I would leave a comment here or somewhere close, something like
```
// as per commonmark spec, there can be any number of backticks, not only one or three
```
otherwise someone (like me) might think to optimize this to `indexOf("```")` or `indexOf("`")`

then your other comment would make more sense actually :+1: